protobuf2.5/plan.sh

```diff
@@ -0,0 +1,14 @@
+pkg_name=protobuf
```

Comment:
@mdrobnak on this one we're going to have to change it. The `pkg_name` should be `protobuf2.5`, and you can set a `pkg_distname` to `protobuf`. See how [the bison2 plan](https://github.com/habitat-sh/core-plans/blob/2c521b27177027023f92bcd41f25bfcaf1dd6e1a/bison2/plan.sh) works, for example.

I'll open a PR to do that right now, and I'll modify protobuf-c to make it work with this one.